src/gui/qgsmaptooledit.h

```diff
@@ -74,6 +74,14 @@ class GUI_EXPORT QgsMapToolEdit: public QgsMapTool
      */
     int addTopologicalPoints( const QVector<QgsPointXY> &geom );
 
+    /**
+     * Adds vertices to other features to keep topology up to date, e.g. to neighbouring polygons.
+     * \param geom list of points (in layer coordinate system)
+     * \returns 0 in case of success
```

Comment:
What about the other return values? They need describing here, or ideally changed to an explicit enum value

Reply:
Done with enum